### DISCUSSION TITLE
Media Query

### DISCUSSION THREAD

**Discussion by matt1208** [chakra-ui/chakra-ui#3174] [Q&A] [answered]
I am currently using a Stack, to make a side navigation bar, if there a way for when the screen is about less than 1000px to utilize a Drawer, and put the stack in the drawer, so the user can click on a button to access the side nav? I have the code for both the Stack and Drawer separate but I am not sure how to make it responsive so they switch when the screen shrinks.

**Comment by chr-ge** [comment, marked as answer]
You can use the [useMediaQuery](https://chakra-ui.com/docs/hooks/use-media-query#usage) hook
```jsx
import { useMediaQuery } from "@chakra-ui/react"
```

**Reply by matt1208** [reply]
@chr-ge Would you be able to show an example of how to achieve this with switching between a stack and a drawer? I have read through the docs but it only shows how to switch between different texts

**Reply by chr-ge** [reply]
Here is a quick example: https://codesandbox.io/s/still-mountain-ircfo?file=/src/index.js

**Reply by matt1208** [reply]
Thanks that makes more sense!
